public/app/features/variables/inspect/VariablesUnknownTable.test.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import * as runtime from '@grafana/runtime';
+import { render, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { VariablesUnknownTable, VariablesUnknownTableProps } from './VariablesUnknownTable';
+import { customBuilder } from '../shared/testing/builders';
+import * as utils from './utils';
+import { UsagesToNetwork } from './utils';
+
+async function getTestContext(
+  overrides: Partial<VariablesUnknownTableProps> | undefined = {},
+  usages: UsagesToNetwork[] = []
+) {
+  jest.clearAllMocks();
+  const reportInteractionSpy = jest.spyOn(runtime, 'reportInteraction').mockImplementation();
+  const getUnknownsNetworkSpy = jest.spyOn(utils, 'getUnknownsNetwork').mockResolvedValue(usages);
+  const defaults: VariablesUnknownTableProps = {
+    variables: [],
+    dashboard: null,
+  };
+  const props = { ...defaults, ...overrides };
+  const { rerender } = render(<VariablesUnknownTable {...props} />);
+  await waitFor(() =>
+    expect(screen.getByRole('heading', { name: /renamed or missing variables/i })).toBeInTheDocument()
+  );
+
+  return { reportInteractionSpy, getUnknownsNetworkSpy, rerender };
+}
+
+describe('VariablesUnknownTable', () => {
+  describe('when rendered', () => {
+    it('it should render the section header', async () => {
```

Comment:
mega minor nit, don't think we need the `it` at the start of every test

Reply:
Again, I usually write `then it` but I missed it. Thanks for noticing.

Reply:
Fixed!